hrpc/mutate.go

```diff
@@ -100,6 +100,9 @@ func TimestampUint64(ts uint64) func(Call) error {
 		if !ok {
 			return errors.New("'TimestampUint64' option can only be used with mutation queries")
 		}
+		if ts > MaxTimestamp {
```

Comment:
(GitHub is not allowing me to comment on lines that are not part of the diff, so I'm posting it here.)

In `Timestamp` you should also check the time. It's possible that the user passes in a time.Time that is before the unix epoch, which will end up being negative / a value greater than MaxTimestamp when converted to uint64.